dask_sql/physical/rel/logical/aggregate.py

```diff
@@ -61,6 +61,21 @@ def __init__(self, numerical_aggregation, non_numerical_aggregation=None):
             non_numerical_aggregation or numerical_aggregation
         )
 
+    def get_dtype_relevant_aggregation(self, dtype):
+        numeric_agg = self.numerical_aggregation
+
+        if pd.api.types.is_numeric_dtype(dtype):
+            return numeric_agg
+
+        # Todo: Add Categorical when support comes to dask-sql
```

Comment:
Have not added Categorical as thats not supported currently with dask-sql . 

`string` seems to work as expected for these three aggregations.